app/controllers/admin/videos_controller.rb

```diff
@@ -17,8 +17,8 @@ def create
       video.save
 
       flash[:success] = 'Successfully created video.'
-    rescue # Sorry about this. We should get more specific instead of swallowing all errors.
-      flash[:error] = 'Unable to create video.'
+    rescue StandardError
+      flash[:error] = video.errors.full_messages.to_sentence
```

Comment:
This would fix the Rubocop error but we have nothing currently to test it. RSpec runs with ~70% coverage

Reply:
Great job figuring out how to fix the rubocop errors!